export/cloud_datastore_admin.py

```diff
@@ -12,107 +12,100 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Request handler for exporting backups.
+"""Configures a request handler for exporting backups."""
 
-Please see original reference here:
-
-https://cloud.google.com/datastore/docs/schedule-export
-"""
-
-from __future__ import absolute_import  # pylint: disable=import-only-modules
-from __future__ import unicode_literals  # pylint: disable=import-only-modules
+from __future__ import absolute_import # pylint: disable=import-only-modules
+from __future__ import unicode_literals # pylint: disable=import-only-modules
 
 import datetime
-import http.client
-import json
 import logging
 
-from export import acl_decorators
+from core.controllers import base
```

Comment:
I did a test run of this and got:

```
Traceback (most recent call last):
  File "/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py", line 238, in Handle
    handler = _config_handle.add_wsgi_middleware(self._LoadHandler())
  File "/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py", line 309, in _LoadHandler
    handler, path, err = LoadObject(self._handler)
  File "/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py", line 83, in LoadObject
    obj = __import__(path[0])
  File "/cloud_datastore_admin.py", line 23, in <module>
    from core.controllers import base
ImportError: No module named core.controllers
```

I think you'll need to treat this as a separate app that's not connected with base at all...

Reply:
Done.